### DISCUSSION TITLE
Command Palette

### DISCUSSION THREAD

**Discussion by troux** [community/community#16040] [Other Feature Feedback, Questions, & Ideas]
It looks nice to use, but override chrome shortcut which is annoying :disappointed:

**Comment by wilhelmy** [comment]
Heavily dislike this feature because this keybinding is bound to the web search in the browser. Not sure why websites can even override it, but they definitely shouldn't :)

Edit: I wouldn't mind the feature if github left ctrl+k alone

**Comment by queenofcorgis** [comment]
Hi there 👋🏼, Thanks for the feedback, we do have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.

**Reply by wilhelmy** [reply]
Good to hear, but unfortunately an extra step to restore a behaviour that is a default everywhere else in my browser.

**Reply by csrl** [reply]
the github ux folks:  let's take over the default search key binding for our own search because people will be like, wow that's great, I can search github specific stuff, so convenient.   To borrow 90's vernacular:  NOT!

**Comment by kioviensis** [comment]
I like this feature so much but it became so laggy with loading for the last few weeks :c

**Comment by neilgoldman** [comment]
Both Sublime and VSCode use cmd+shift+p / ctrl+shift+p for the command palette. I don't get why Github didn't stick with that convention.
